docs/installation/native-installation.md

```diff
@@ -1,109 +1,27 @@
 
-*This installation guide is not tested.*
+After you read the articles you are ready to start with the creating and building the Shopsys Framework project.
 
-## Requirements
-First of all, you need to install the following software on your system:
+## Create new project from Shopsys Framework sources
 
-### Linux / MacOS
-* [GIT](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
-* [PostgreSQL 9.5](https://wiki.postgresql.org/wiki/Detailed_installation_guides)
-* [PHP 7.1 - 7.3](http://php.net/manual/en/install.php) (configure your `php.ini` by [Required PHP Configuration](../introduction/required-php-configuration.md))
-* [Composer](https://getcomposer.org/doc/00-intro.md#globally)
-* [Node.js with npm](https://nodejs.org/en/download/) (npm is automatically installed when you install Node.js)
-* [Redis](https://redis.io/topics/quickstart)
-* [Elasticsearch](https://www.elastic.co/guide/en/elasticsearch/reference/current/install-elasticsearch.html)
-
-### Windows
-* [GIT](https://git-scm.com/download/win)
-* [PostgreSQL 9.5](https://www.enterprisedb.com/downloads/postgres-postgresql-downloads#windows)
-* [PHP 7.1 - 7.3](http://php.net/manual/en/install.windows.php) (configure your `php.ini` by [Required PHP Configuration](../introduction/required-php-configuration.md))
-* [Composer](https://getcomposer.org/doc/00-intro.md#installation-windows)
-* [Node.js with npm](https://nodejs.org/en/download/) (npm is automatically installed when you install Node.js)
-* [Redis](https://github.com/MicrosoftArchive/redis/releases)
-* [Elasticsearch](https://www.elastic.co/guide/en/elasticsearch/reference/current/install-elasticsearch.html)
-
-*Note: The names link to the appropriate installation guide or download page.*
-
-***Note:** During installation there will be installed 3-rd party software as dependencies of Shopsys Framework by [Dockerfile](https://docs.docker.com/engine/reference/builder/), [composer](https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies) and [npm](https://docs.npmjs.com/about-the-public-npm-registry) with licenses that are described in document [Open Source License Acknowledgements and Third-Party Copyrights](../../open-source-license-acknowledgements-and-third-party-copyrights.md)*
-
-After that, you can follow the steps below in order to install and configure Shopsys Framework.
-
-## Steps
-### 1. Create new project from Shopsys Framework sources
 ```
-composer create-project shopsys/project-base --keep-vcs
-cd project-base
+php -d memory_limit=-1 <PATH TO COMPOSER or COMPOSER.PHAR> create-project shopsys/project-base --keep-vcs
 ```
 
 *Notes:*
 - *The `--keep-vcs` option initializes GIT repository in your project folder that is needed for diff commands of the application build and keeps the GIT history of `shopsys/project-base`.*
+- *Since `v7.0.0-beta4` we have set memory limit to `-1` for composer because of the increased memory consumption during the dependencies calculation.*
```

Comment:
Do we really want to have in docs what happened since the beta stage?

Reply:
first 2 words removed.